README.md

```diff
@@ -1,115 +1,101 @@
 * `BUILD_MACHINE_USERNAME` — your build machine user. Something like `john_doe`.
 
 # Authentication
 
-### Configure ssh
+### Configure SSH
 
-  * Generate new ssh key or use existing one.
+1. Generate SSH key.
 
   ```
   $ ssh-keygen -t rsa -b 4096 -C "{BUILD_MACHINE_USERNAME}"
   ```
 
-  * Run the command
-  * Enter file location for new key, ie: `~/.ssh/remote-build`.
-  
-
-  * Append following content to `~/.ssh/config`.
+2. Append the following content to `~/.ssh/config`.
 
   ```config
   Host {BUILD_MACHINE_NAME}
     User {BUILD_MACHINE_USERNAME}
     HostName {BUILD_MACHINE_IP_OR_HOSTNAME}
-    IdentityFile ~/.ssh/remote-build
+    IdentityFile ~/.ssh/{SSH_KEY_NAME}
     PreferredAuthentications publickey
     ControlMaster auto
     ControlPath /tmp/%r@%h:%p
     ControlPersist 1h
   ```
 
-  * Specifying `PreferredAuthentications` speeds up connection phase.
-  * `ControlMaster` allows reusing existing ssh connections.
-  * `ControlPersist` specifies for how long ssh should keep connection open.
+  * `ControlMaster` enables SSH connection reusage.
+  * `ControlPersist` specifies for how long SSH should keep connection open.
 
-* Copy and send public key to the person responsible for maintenance of remote build infrastructure.
+3. Copy and send public key to the person responsible for maintenance of remote build infrastructure.
 
-  ```
-  # On macOS you can copy content from commandline.
+  ```shell
+  # macOS-specific. Linux users, you know what to do.
   $ pbcopy < ~/.ssh/sshkey.pub
   ```
 
-* Once you've received confirmation that build machine is ready for you, test the connection.
+4. Once you’ve received confirmation that build machine is ready for you, test the connection.
 
   ```
   $ ssh {BUILD_MACHINE_HOSTNAME}
   ```
 
 # Local Configuration
 
-Download [`mainframer.sh`](mainframer.sh) and save it in your project (we recommend to put it under version control so you could sync changes across all team members).
-
-Also:
->We recommend you subscribe to changes in this repo somehow (follow it on GitHub / watch for tweets of its maintainers / etc), this will allow you always apply best practises we found to make your Remote Build better, faster and safer.
+1. Download [`mainframer.sh`](mainframer.sh) and save it in your project, most likely put it to VCS so you could sync changes across all team members.
 
-Put the following content in your local `local.properties` file.
+  We recommend you to subscribe to changes in this repo somehow (follow it on GitHub / watch for tweets of its maintainers / etc). This will allow you to apply best practises we found to make your Remote Build better, faster and safer.
 
-```properties
-# Required.
-remote_build.machine={BUILD_MACHINE_NAME}
-```
+2. Update your `local.properties` file.
 
-That'll be passed to `ssh` as parameter, `user@machine` or `machine` or `ip` will be ok (depending on your ssh and remote machine config of course).
```

Comment:
We introduce `BUILD_MACHINE_NAME` just before and use it afterwards as a credential (kind of) to put to SSH config. After putting it to config it should be used there.  Not sure it is necessary to repeat ourselves.